app/_locales/en/messages.json

```
@@ -44,6 +44,18 @@
   "QRHardwareWalletSteps2Description": {
     "message": "Ngrave Zero"
   },
+  "SrpListHideAccounts": {
+    "message": "Hide $1 accounts"
+  },
+  "SrpListHideSingleAccount": {
+    "message": "Hide $1 account"
+  },
+  "SrpListShowAccounts": {
+    "message": "Show $1 accounts"
+  },
+  "SrpListShowSingleAccount": {
+    "message": "Show $1 account"
+  },
```

Small suggestion, I think you can drop the variable for the single case since it is just one. And also added a short description to the others

```suggestion
  "SrpListHideAccounts": {
    "message": "Hide $1 accounts",
    "description": "$1 is the number of accounts"
  },
  "SrpListHideSingleAccount": {
    "message": "Hide 1 account"
  },
  "SrpListShowAccounts": {
    "message": "Show $1 accounts",
    "description": "$1 is the number of accounts"
  },
  "SrpListShowSingleAccount": {
    "message": "Show 1 account"
  },
```